andela/ah-jarvis-frontend
# #159961943 Setup Redux

#### What does this PR do?
Setup redux and folder structure for the app

#### Description of Task to be completed?
- Install redux and react-redux libraries
- Setup store and rootReducer
- Create an example container 
- Connect redux to the app via Provider

#### How should this be manually tested?
- Clone this branch

```
git clone -b  ch-redux-setup-159961943 git@github.com:andela/ah-jarvis-frontend.git

cd ah-jarvis-frontend
  
npm install or yarn

npm start or yarn start

```

You should get the following output on your browser.

<img width="1280" alt="screen shot 2018-08-30 at 15 37 24" src="https://user-images.githubusercontent.com/22371596/44852030-db445700-ac6a-11e8-979a-7406cafa2289.png">


#### What are the relevant pivotal tracker stories?
[#159961943](https://www.pivotaltracker.com/story/show/159961943)
